lib/registries/addon/drafts/draft/-components/register/template.hbs

```diff
@@ -0,0 +1,43 @@
+<OsfButton
+    data-test-goto-register
+    @type='success'
+    @onClick={{action (mut this.showPartialRegDialog) true}}
+    @disabled={{not this.draftManager.registrationResponsesIsValid}}
+>
+    {{t 'registries.drafts.draft.register'}}
+</OsfButton>
+
+{{#unless this.draftManager.registrationResponsesIsValid}}
+    <div data-test-invalid-responses-text class='text-danger'>
+        {{t 'registries.drafts.draft.review.invalid_warning'}}
+    </div>
+{{/unless}}
+
+{{#if this.rootNode}}
+    <Registries::PartialRegistrationModal::Manager
+        @rootNode={{this.rootNode}}
+        as |partialRegistrationManager|
+    >
+        <Registries::PartialRegistrationModal
+            @isOpen={{this.showPartialRegDialog}}
+            @manager={{partialRegistrationManager}}
+            @onContinue={{action this.onContinue}}
+            @onClose={{action (mut this.showPartialRegDialog) false}}
```

Comment:
This should probably be an action. Instead of toggling state directly, it might make sense to create these actions:
```ts
@action
showPartialRegDialog() {
    this.set('finalizeRegDialogIsOpen', false);
    this.set('partialRegDialogIsOpen', true);
}

@action
showFinalizeRegDialog() {
    this.set('partialRegDialogIsOpen', false);
    this.set('finalizeRegDialogIsOpen', true);
}

@action
closeAllDialogs() {
    this.set('partialRegDialogIsOpen', false);
    this.set('finalizeRegDialogIsOpen', false);
}
```
and use those here and in `onContine`, `onBack`, `onSubmit`.